src/Payment.cs

```diff
@@ -232,6 +232,20 @@ public Payment ValidateUpi(Dictionary<string, object> data)
                 return null;
             }
         }
+
+        public Payment ExpandedDetails(Dictionary<string, object> data)

```

Comment:
It could be an enhancement, but shall we add any validation on 'expand[]' value.